utils/doclint/documentation.js

```diff
@@ -278,8 +278,9 @@ Documentation.Member = class {
    * @param {MarkdownNode[]=} spec
    * @param {boolean=} required
    * @param {string[]=} templates
+   * @param {number} order
    */
-  constructor(kind, langs, name, type, argsArray, spec = undefined, required = true, templates = []) {
+  constructor(kind, langs, name, type, argsArray, spec = undefined, required = true, templates = [], order = null) {
```

Comment:
Is `order` ever set to anything but null? if not, just drop it from the argument list